cinema/views.py

```diff
@@ -167,6 +199,25 @@ def get_serializer_class(self):
 
         return MovieSessionSerializer
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                name="date",
+                description="Filter by date (?date=2024-12-31)",
+                type=str,
+            ),
+            OpenApiParameter(
+                name="movie",
+                description="Filter by movie id (?movie_id=3,5,8,13)",
+                type=int,
+            ),
+        ],
+        description="Retrieve a list of movie sessions. "
+                    "You can filter sessions by date and by movie_id.",
+    )
+    def list(self, request, *args, **kwargs):
```

Comment:
The `list` method is correctly overridden to include the `@extend_schema` decorator, ensuring that the API documentation is generated for this endpoint.